<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinDialog</class>
 <widget class="QDialog" name="JoinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Join items</string>
  </property>
  <property name="windowIcon">
   <iconset theme="media-playlist-repeat">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="classLabel">
     <property name="text">
      <string>Class</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="Class">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>&lt;p&gt;Class of items to join geometry.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QListWidget" name="Values">
     <property name="toolTip">
      <string>&lt;p&gt;Attributes to check&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar"/>
   </item>
   <item row="3" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="Keep">
       <property name="toolTip">
        <string>If enabled keeps oryginal objects and marks them as removed.</string>
       </property>
       <property name="text">
        <string>Keep changed objects</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="procedButton">
       <property name="text">
        <string>Join</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>procedButton</sender>
   <signal>clicked()</signal>
   <receiver>JoinDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>363</x>
     <y>405</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>JoinDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onListRequest(QString)</slot>
 </slots>
</ui>
